modules/AOW_WorkFlow/controller.php

```diff
@@ -549,7 +549,7 @@ protected function action_getAction()
 
         $id = '';
         $params = array();
-        if (isset($_REQUEST['id'])) {
+        if (isset($_REQUEST['id']) && strlen(trim($_REQUEST['id'])) > 0 ) {
```

Comment:
Hi @markbond1007, would `!empty($_REQUEST['id']))` do the same as `isset($_REQUEST['id']) && strlen(trim($_REQUEST['id'])) > 0`?

Reply:
@clemente-raposo 

Yes that would work just as well (just tried it and its fine).

Mark